gapis/trace/android/adreno/profiling_data.go

```diff
@@ -90,6 +90,45 @@ func extractTraceHandles(ctx context.Context, replayHandles *[]int64, replayHand
 	}
 }
 
+func fixContextIds(contextIDs []int64) []int64 {
+	// This is a temporary work to workaround a QC bug
```

Comment:
If this is temporary, we should put some context here to make it easy to remove when appropriate. Do we have a tracking bug for the underlying fix? Is there any kind of timeline for when this would be fixed? I assume this would still be an issue on older/not updated devices for a while? How would we validate that the fix is not necessary anymore? 

The answers should not necessarily be in the comment here, but probably a link to a resource that has more of this context to follow up.